src/SortingAndFiltering/sortByPrice.js

```diff
@@ -0,0 +1,11 @@
+function sortByPrice(sortBy, data) {
+  let newData = [...data];
+  if (sortBy === "HIGH_TO_LOW")
+    return newData.sort((a, b) => b["price"] - a["price"]);
```

Comment:
no need to create another variable her eyou can just write [...data].sort()

Reply:
done